coderdojochi/settings.py

```diff
@@ -231,19 +234,22 @@ class MediaRootS3BotoStorage(S3Boto3Storage):
         def __init__(self):
             super().__init__(location='media', file_overwrite=False)
 
-    # endregion
```

Comment:
If `endregion` is removed, you should remove the `region` comment on line 226 as well.